test/e2e_node/swap_test.go

```diff
@@ -56,8 +56,9 @@ var (
 	noLimits *resource.Quantity = nil
 )
 
-var _ = SIGDescribe("Swap", "[LinuxOnly]", nodefeature.Swap, func() {
+var _ = SIGDescribe("Swap", "[LinuxOnly]", nodefeature.Swap, framework.WithSerial(), func() {
```

Comment:
IIRC this was not made serial on purpose because the test mostly reads system properties. But in our case, running pods changes the system state so for this reason is better to make it `Serial` indeed

Reply:
> IIRC this was not made serial on purpose because the test mostly reads system properties

That's right!

> But in our case, running pods changes the system state so for this reason is better to make it Serial indeed

I'd be happy for some clarifications here :)
What do you mean by "our case"? Is every test that's running pods have to be Serial?